app/src/main/java/at/roteskreuz/stopcorona/screens/reporting/reportStatus/ReportingStatusFragment.kt

```diff
@@ -155,4 +174,25 @@ class ReportingStatusFragment : BaseFragment(R.layout.fragment_reporting_status)
         viewModel.goBack()
         return true // the changing of fragments is managing parent activity
     }
+    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
+        super.onActivityResult(requestCode, resultCode, data)
+        when (requestCode) {
+            ExposureNotificationRepository.DebugAction.REGISTER_WITH_FRAMEWORK.requestCode() -> {
```

Comment:
Same as in debug fragment, please refactor to have request code as a responsibility of View Layer.